tests/run.sh

```diff
@@ -174,6 +174,9 @@ if [ -n "$DO_COVERAGE" ] && [ "$DO_COVERAGE" != 0 ]; then
     sed "1 s~^~require('luacov.runner')('$source_dir/.luacov'); \0~" \
         "$RC_FILE" > "$tmp_files/awesomerc.lua"
     RC_FILE=$tmp_files/awesomerc.lua
+
+    # Use xresources theme for more coverage.
+    sed -i 's~default/theme~xresources/theme~' "$RC_FILE"
```

Comment:
Nope, you cannot restart awesome between tests. After the restart, awesome would not know that it is supposed to continue executing some text. Also, changing the theme at runtime is not a good idea since nothing will use the changed theme values for anything.

If you just want to test that the themes themselves contain valid Lua code, you can `dofile()` them.